editor/editor_resource_picker.cpp

```diff
@@ -283,6 +283,7 @@ void EditorResourcePicker::_update_menu_items() {
 
 		if (paste_valid) {
 			edit_menu->add_item(TTR("Paste"), OBJ_MENU_PASTE);
+			edit_menu->add_item(TTR("Paste as Unique"), OBJ_MENU_PASTE_AS_UNIQUE);
```

Comment:
```suggestion
			edit_menu->add_item(TTRC("Paste as Unique"), OBJ_MENU_PASTE_AS_UNIQUE);
```
Preferred for new messages.